dovecot/usr/local/lib/templates/local.conf

```
@@ -108,7 +116,24 @@ service stats {
 # 90-sieve
 #
 plugin {
-  sieve_plugins = # sieve_extprograms sieve_imapsieve
+  sieve_plugins = sieve_extprograms sieve_imapsieve
+  sieve_global_extensions = +vnd.dovecot.pipe +vnd.dovecot.environment +editheader
+  sieve_editheader_max_header_size = 2048
+  sieve_before = /etc/dovecot/spam-actions.sieve
```

```suggestion
  sieve_after = /etc/dovecot/spam-actions.sieve
```
Allow user filters to override junk filing